{
  "uid" : "7c235b44f87eba9b",
  "name" : "Verify redirection to used resources page not authorized user.",
  "fullName" : "tests.main_page_test.TestMainPage#test_mp_08_verify_redirection_to_used_resources_page",
  "historyId" : "43df35c0ef3ab6719310f8b9b9ee7530",
  "time" : {
    "start" : 1736560726255,
    "stop" : 1736560766688,
    "duration" : 40433
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
  "statusTrace" : "self = <pages.main_page.MainPage object at 0x7f6ebc67ad80>\nurl = 'https://brainup.site/used-resources'\n\n    def check_expected_link(self, url):\n        with allure.step(f'Check url is present: {url}'):\n            try:\n>               return Wait(self.driver, self.timeout).until(\n                    ec.url_to_be(url), message=f\"Can't find element by locator {url}\")\n\npages/base_page.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"996b64058ab3dcdfaaa961b251b9a425\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f6ebc687f60>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.main_page_test.TestMainPage object at 0x7f6ebc82d6a0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"996b64058ab3dcdfaaa961b251b9a425\")>\nmain_page_open = None\n\n    @allure.title(\"Verify redirection to used resources page not authorized user.\")\n    def test_mp_08_verify_redirection_to_used_resources_page(self, driver, main_page_open):\n        page = MainPage(driver)\n>       resources_page_url = page.open_used_resources_page()\n\ntests/main_page_test.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:68: in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\npages/base_page.py:72: in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"996b64058ab3dcdfaaa961b251b9a425\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f6ebc687740>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "main_page_open",
    "time" : {
      "start" : 1736560722728,
      "stop" : 1736560726255,
      "duration" : 3527
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1736560722728,
        "stop" : 1736560726255,
        "duration" : 3527
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"996b64058ab3dcdfaaa961b251b9a425\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1736560721984,
      "stop" : 1736560722727,
      "duration" : 743
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1736559777376,
      "stop" : 1736559777376,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
    "statusTrace" : "self = <pages.main_page.MainPage object at 0x7f6ebc67ad80>\nurl = 'https://brainup.site/used-resources'\n\n    def check_expected_link(self, url):\n        with allure.step(f'Check url is present: {url}'):\n            try:\n>               return Wait(self.driver, self.timeout).until(\n                    ec.url_to_be(url), message=f\"Can't find element by locator {url}\")\n\npages/base_page.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"996b64058ab3dcdfaaa961b251b9a425\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f6ebc687f60>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.main_page_test.TestMainPage object at 0x7f6ebc82d6a0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"996b64058ab3dcdfaaa961b251b9a425\")>\nmain_page_open = None\n\n    @allure.title(\"Verify redirection to used resources page not authorized user.\")\n    def test_mp_08_verify_redirection_to_used_resources_page(self, driver, main_page_open):\n        page = MainPage(driver)\n>       resources_page_url = page.open_used_resources_page()\n\ntests/main_page_test.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:68: in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\npages/base_page.py:72: in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"996b64058ab3dcdfaaa961b251b9a425\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f6ebc687740>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
    "steps" : [ {
      "name" : "Open used resources page",
      "time" : {
        "start" : 1736560726255,
        "stop" : 1736560766687,
        "duration" : 40432
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/main_page.py\", line 68, in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Check url is present: https://brainup.site/used-resources",
        "time" : {
          "start" : 1736560726431,
          "stop" : 1736560766687,
          "duration" : 40256
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n",
        "statusTrace" : "  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 1,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "f6d6768f7bcca8b5",
      "name" : "Screenshot 2025-01-11 01:59:26.792793",
      "source" : "f6d6768f7bcca8b5.png",
      "type" : "image/png",
      "size" : 58734
    } ],
    "parameters" : [ ],
    "stepsCount" : 2,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1736560766794,
      "stop" : 1736560766864,
      "duration" : 70
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Main Page."
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "main_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestMainPage"
  }, {
    "name" : "host",
    "value" : "fv-az787-514"
  }, {
    "name" : "thread",
    "value" : "2375-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.main_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 19,
        "skipped" : 0,
        "passed" : 78,
        "unknown" : 0,
        "total" : 97
      },
      "items" : [ {
        "uid" : "982f85261286a508",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/150//#testresult/982f85261286a508",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736474440159,
          "stop" : 1736474480624,
          "duration" : 40465
        }
      }, {
        "uid" : "cedc963273834e35",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/149//#testresult/cedc963273834e35",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736389516174,
          "stop" : 1736389556636,
          "duration" : 40462
        }
      }, {
        "uid" : "cdc44bb285a6288",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/148//#testresult/cdc44bb285a6288",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736301540196,
          "stop" : 1736301580668,
          "duration" : 40472
        }
      }, {
        "uid" : "5936a589a76f6bcd",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/147//#testresult/5936a589a76f6bcd",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736215136253,
          "stop" : 1736215176702,
          "duration" : 40449
        }
      }, {
        "uid" : "e579c1ab0992e77",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/146//#testresult/e579c1ab0992e77",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736128959107,
          "stop" : 1736128999584,
          "duration" : 40477
        }
      }, {
        "uid" : "fe48b61786378ad0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/145//#testresult/fe48b61786378ad0",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736042707551,
          "stop" : 1736042748020,
          "duration" : 40469
        }
      }, {
        "uid" : "bc9f5e7c213334df",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/144//#testresult/bc9f5e7c213334df",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735955851081,
          "stop" : 1735955891500,
          "duration" : 40419
        }
      }, {
        "uid" : "acb8e1f1f5888930",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/143//#testresult/acb8e1f1f5888930",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735869521718,
          "stop" : 1735869562221,
          "duration" : 40503
        }
      }, {
        "uid" : "c6eda87d073df44b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/142//#testresult/c6eda87d073df44b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735783067778,
          "stop" : 1735783108255,
          "duration" : 40477
        }
      }, {
        "uid" : "48d7fbff38251f05",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/141//#testresult/48d7fbff38251f05",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735697122874,
          "stop" : 1735697163357,
          "duration" : 40483
        }
      }, {
        "uid" : "aef6fee2b589b392",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/140//#testresult/aef6fee2b589b392",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735610352647,
          "stop" : 1735610393098,
          "duration" : 40451
        }
      }, {
        "uid" : "466a4c5edbcb31a6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/139//#testresult/466a4c5edbcb31a6",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735524091416,
          "stop" : 1735524131890,
          "duration" : 40474
        }
      }, {
        "uid" : "ac3d56de29e25d0c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/138//#testresult/ac3d56de29e25d0c",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735437966168,
          "stop" : 1735438006624,
          "duration" : 40456
        }
      }, {
        "uid" : "869668786dbf1231",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/137//#testresult/869668786dbf1231",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735350941549,
          "stop" : 1735350982032,
          "duration" : 40483
        }
      }, {
        "uid" : "7da3b9bd38c86f56",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/136//#testresult/7da3b9bd38c86f56",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1735264786093,
          "stop" : 1735264826540,
          "duration" : 40447
        }
      }, {
        "uid" : "b12eea9e482ab0b0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/81//#testresult/b12eea9e482ab0b0",
        "status" : "passed",
        "time" : {
          "start" : 1730512032351,
          "stop" : 1730512032501,
          "duration" : 150
        }
      }, {
        "uid" : "18a5185b222d833c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/80//#testresult/18a5185b222d833c",
        "status" : "passed",
        "time" : {
          "start" : 1730426208208,
          "stop" : 1730426208355,
          "duration" : 147
        }
      }, {
        "uid" : "96aba1fd8a508c36",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/79//#testresult/96aba1fd8a508c36",
        "status" : "passed",
        "time" : {
          "start" : 1730339452369,
          "stop" : 1730339452518,
          "duration" : 149
        }
      }, {
        "uid" : "5258f23c57c5c837",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/78//#testresult/5258f23c57c5c837",
        "status" : "passed",
        "time" : {
          "start" : 1730252960892,
          "stop" : 1730252961058,
          "duration" : 166
        }
      }, {
        "uid" : "d87bea3b032e3df9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/77//#testresult/d87bea3b032e3df9",
        "status" : "passed",
        "time" : {
          "start" : 1730166658408,
          "stop" : 1730166658569,
          "duration" : 161
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "7c235b44f87eba9b.json",
  "parameterValues" : [ ]
}